<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MadmGui_Total_Rank_Wid</class>
 <widget class="QWidget" name="MadmGui_Total_Rank_Wid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>584</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="head_label">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Ranking (Nobreak risk approach)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_id">
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>479</width>
        <height>842</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Ranking of the measures's mid-value of the ranks for all four MADM-approaches</string>
         </property>
         <property name="title">
          <string>Total</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="MadmGui_Rank_Wid" name="tableWidget_total"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="title">
          <string>SAW (Standard)</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="MadmGui_Rank_Wid" name="tableWidget_saw_standard"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="title">
          <string>SAW (Range)</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="MadmGui_Rank_Wid" name="tableWidget_saw_range"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="title">
          <string>TOPSIS</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="MadmGui_Rank_Wid" name="tableWidget_topsis"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="title">
          <string>ELECTRE</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="MadmGui_Rank_Wid" name="tableWidget_electre"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MadmGui_Rank_Wid</class>
   <extends>QTableWidget</extends>
   <header>madmgui_rank_wid.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
